crane/cmd.go

```diff
@@ -2,9 +2,10 @@ package crane
 
 import (
 	"fmt"
-	"github.com/alecthomas/kingpin"
 	"os"
 	"strings"
+
+	"github.com/alecthomas/kingpin"
```

Comment:
Ah okay - then that's all good. I wanted to get that running a while back but for unclear reasons it did not work ... have to try that again, looks simple to setup.